packages/server/lib/browsers/cri-client.ts

```diff
@@ -86,7 +140,21 @@ export const initCriClient = async (debuggerUrl: websocketUrl): Promise<CRIWrapp
         throw err
       })
     },
-
+    takeScreenshot: () => {
+      return ensureMinimumProtocolVersion('1.3')
+      .catch((err) => {
+        throw new Error(`Taking a screenshot requires at least Chrome 64.\n\nDetails:\n${err.message}`)
```

Comment:
We should probably just throw even earlier if the version is < 1.3, since `Network.setCookie` (required for Cypress to work) is only available starting with 1.3.

Reply:
7fd44a200670d407c39246ac7dacf41c15347702